critiquebrainz/frontend/templates/macros.html

```diff
@@ -92,3 +92,11 @@
     {% endif %}
   </div>
 {%- endmacro %}
+
+{% macro soundcloud_player(release_group_id, soundcloud_mapping) %}
```

Comment:
And `soundcloud_mapping` would be better named as `url`. It's not really a mapping and it's obvious that it would point to a SoundCloud page.